src/commands/image/imageSource/EnvironmentVariablesListStep.ts

```diff
@@ -29,6 +29,8 @@ export class EnvironmentVariablesListStep extends AzureWizardPromptStep<ImageSou
 
     public async configureBeforePrompt(context: ImageSourceBaseContext): Promise<void> {
         if (context.environmentVariables?.length === 0) {
+            context.telemetry.properties.environmentVariableFileCount = '0';
+            context.telemetry.properties.setEnvironmentVariableOption = SetEnvironmentVariableOption.NoDotEnv;
             this.outputLogs(context, SetEnvironmentVariableOption.NoDotEnv);
         }
     }
```

Comment:
Conversely, it may be valuable to set the count if it's more than 0.

Reply:
Agree, I ended up logging all the files based on glob patterns searched under `selectWorkspaceFile`:

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/7d308f08-180a-4f84-ab19-5d1f28eae94a)

Hopefully that's an okay way to handle this